Source/Scene/Cesium3DTileset.js

```diff
@@ -1617,6 +1634,34 @@ define([
 
         // Number of commands added by each update above
         tileset._statistics.numberOfCommands = (commandList.length - numberOfInitialCommands);
+
+        if (tileset.debugShowGeometricError) {
```

Comment:
Move the code inside this `if` block into their own function so this function flows better.  In general, each statement in a function should be roughly at the same level of abstraction.